samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -2677,4 +2699,3 @@ const struct bt_mesh_comp comp = {
 	.elem = elements,
 	.elem_count = ARRAY_SIZE(elements),
 };
-
```

Comment:
How is this change related to the commit?

Reply:
I think it was get automatically removed because of using atom editor. 
I've added line as it was. So done !!